### ISSUE TITLE
About max resolution and the number of bounding box

### ISSUE THREAD

**Issue by qinax-v** [yu-rp/VisualPerceptionToken#11]
Thanks for your open work.

In the paper, the performance is constrained by max resolution. Why the resolution is considered during inference? Is it possible to use original image resolution during inference?

Another question: does the training data include samples with multiple bounding boxes (sometimes, the answer is generated by viewing multiple image regions)?

Thank you~

**Comment by yu-rp** [comment]
Hi @xwqian , 

Thank you for your issue. 

The resolution is a key factor, which influencing the performance. Controlling the resolution is quite common in the evaluation of MLLM to ensure fairness. Why not using the original resolution? Because some images are too big to be supported by our GPU so we have to resize them. The question, therefore, is which images should be resized and to what extent. In this paper, we consider two threshold values, such as 512. Specifically, if either the height or width of an image exceeds 512, the image will be resized so that its larger dimension becomes 512, while preserving the aspect ratio. If both dimensions are less than 512, the image is kept at its original resolution.

Thank you for mentioning one important future direction. We do not include samples with multiple bboxes. However, this is certainly an interesting direction to explore.

Runpeng

**Comment by qinax-v** [comment]
Hi, @yu-rp ,

Thanks for your detailed answer. I understand your concerns about the resolution. While I still have a question:

If GPUs are sufficient to support a larger resolution (e.g., 1280), is it possible to infer with the larger resolution and achieve a better performance? Does the larger resolution affect the accuracy of region bounding box when generating Region Selection Token?

Thank you~

**Comment by yu-rp** [comment]
Hi, @xwqian 

1. Yes. With enough GPU memory, inference with larger resolutions can be supported.
2. Probably. Because generally larger resolution leads to more tokens, resulting in better performance. But we did not test under such setup.
